booking/src/main/java/org/dcsa/conformance/standards/booking/model/PersistableCarrierBooking.java

```diff
@@ -93,7 +93,7 @@ public void confirmBookingAmendment(String reference, String reason) {
     changeState(BOOKING_STATUS, CONFIRMED);
     changeState(AMENDED_BOOKING_STATUS, AMENDMENT_CONFIRMED);
     mutateBookingAndAmendment(this::ensureConfirmedBookingHasCarrierFields);
```

Comment:
The `reason` parameter is still unused in this method.

I assume the `setReason(...)` being removed is something you aligned with Henrik. I found it surprising, but if Henrik said so then it is fine.

Reply:
the reason is here an unused parameter. I will remove it . I checked with @HenrikHL (also IFS) that reason field is not present for booking states (the RECEIVED and CONFIRMED).

Reply:
If the `reason` **must not** be present, then I suspect you want `setReason(null)` to ensure it is always cleared (it would also clear the reason field on confirming the booking, which probably makes sense either additionally in that case).

Reply:
it make sense to clear